### PR TITLE
fix: makefile

### PR DESCRIPTION
**Pull request by alanshaw**
Fixes the makefile so a developer can get started _without_ THERE BE DRAGONS.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,12 +4,12 @@ ifneq (,$(wildcard ./.env))
...
-GOARCH=arm64
-GOCC?=go
-GOFLAGS=-tags=lambda.norpc
-CGO_ENABLED=0
-LAMBDAS=build/getclaims/bootstrap build/getroot/bootstrap build/notifier/bootstrap build/postclaims/bootstrap build/providercache/bootstrap build/remotesync/bootstrap
+LAMBDA_GOOS=linux
+LAMBDA_GOARCH=arm64
+LAMBDA_GOCC?=go
+LAMBDA_GOFLAGS=-tags=lambda.norpc
+LAMBDA_CGO_ENABLED=0
```
Comment (alanshaw) [inline Makefile:11]: Overwriting these env vars causes problems when building for local execution i.e. `ucankey`

File: Makefile [hunk 2]
```diff
@@ -4,12 +4,12 @@ ifneq (,$(wildcard ./.env))
...
+LAMBDAS=build/getclaim/bootstrap build/getclaims/bootstrap build/getroot/bootstrap build/notifier/bootstrap build/postclaims/bootstrap build/providercache/bootstrap build/remotesync/bootstrap
```
Comment (alanshaw) [inline Makefile:12]: Added missing `getclaim` lambda.

File: Makefile [hunk 3]
```diff
@@ -38,10 +38,10 @@ lambdas: $(LAMBDAS)
 .PHONY: $(LAMBDAS)
 
 $(LAMBDAS): build/%/bootstrap:
-	GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=$(CGO_ENABLED) $(GOCC) build $(GOFLAGS) -o $@ cmd/lambda/$*/main.go
+	GOOS=$(LAMBDA_GOOS) GOARCH=$(LAMBDA_GOARCH) CGO_ENABLED=$(LAMBDA_CGO_ENABLED) $(LAMBDA_GOCC) build $(LAMBDA_GOFLAGS) -o $@ cmd/lambda/$*/main.go
 
 deploy/.terraform:
-	tofu -chdir=deploy init
+	TF_WORKSPACE= tofu -chdir=deploy init
```
Comment (alanshaw) [inline Makefile:44]: Ok so, tofu expects no existing `TF_WORKSPACE` when initializing, but then needs it subsequently.

File: Makefile (hunk 1 above)
Comment (volmedo) [reply Makefile:11]: yeah, that makes sense as they are the vars Go uses

**Review by volmedo — APPROVED** [review]
LGTM!

**Review by hannahhoward — COMMENTED** [review]
LGTM

**Pull request merged**